www/code/app-code.less

```
@@ -144,7 +144,7 @@
             resize: none;
         }
```

You can add the `max-width` rule here and keep `40vw` above to avoid the second side-effect.
```suggestion
         }
         #cp-app-code-preview-content {
             max-width: 100%;
         }
```